apps/notepad/src/app/component/notes-list/notes-list.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex flex-wrap">
+  <div
+    *ngFor="let note of notesList; let i = index"
+    class="max-w-sm rounded overflow-hidden shadow-lg"
+    (click)="onCardClick()"
```

Comment:
Event binding with onCardClick can be removed after removing the function from typescript file (since it's unused)